extensions/lifecycle/bucketProcessor/LifecycleBucketProcessor.js

```diff
@@ -456,12 +456,18 @@ class LifecycleBucketProcessor {
             done => this._initKafkaBacklogMetrics(done),
             done => this._setupConsumer(done),
             done => {
-                this._locationStatusStream = new LocationStatusStream('lifecycle',
-                    this._mongoConfig,
-                    this._pauseServiceForLocation.bind(this),
-                    this._resumeServiceForLocation.bind(this),
-                    this._log);
-                this._locationStatusStream.start(done);
+                if (this._mongoConfig) {
```

Comment:
note: eventually (once we support pause/resume on s3c), we _will_ need to have an abstraction here, and always create some sort of "locationStatusStream" : listening to mongo (as today in zenko) or to something else (maybe zookeeper?)

but indeed we don't have this now, this will be done later.